client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -543,12 +585,26 @@ export default {
         })
     },
 
+    loadAsyncComponents (hookName) {
```

Comment:
isn't this a generic method which should live in `libs` next to `loadAddonComponents` ??

and then onMounted we only do something like 
```
  this.asyncComponents = loadAsyncComponents(hookName)
```

Reply:
for refs we can acces them like that